NEWS.md

```diff
@@ -1,5 +1,9 @@
 # tinkr (development version)
 
+* unescaped square braces from the source code can now make the roundtrip
+  without extra markup. This is implemented in a new default parameter for
+  `to_xml()` called `unescaped = TRUE`, which indicates that you would like
```

Comment:
should it be called something more specific? `protect_unescaped_square_braces` is long but clear :thinking:

Reply:
I left it at `unescaped` because I was thinking that we can add in [other commonly escaped characters](https://spec.commonmark.org/0.30/#example-14) later. The only reason why I do not do this now is that I need to make sure the math protection is not adversely affected by this feature (e.g. will having existing `asis` nodes break the search for math delimiters?).